Lib/multiprocessing/process.py

```diff
@@ -169,6 +169,12 @@ def is_alive(self):
             _children.discard(self)
             return False
 
+    def is_closed(self):
+        '''
+        Return whether process is closed
+        '''
+        return self._closed
```

Comment:
In fact, I don't want to make things too complicated, because it only needs to return to the closed state.

Reply:
Yes, and that's the reason why I think a property is preferred over a method, so that the process class has a similar interface as the connection class. I don't see why we need a *method* and not a *property*.

Reply:
Hmmm, I'm on the fence. It seems like this makes sense, and it would actually be good to change that property to public (perhaps internal state should be kept as much as possible and exposed via getting rather than exposing private attribute directly).

Reply:
No. The property should not be public. Simply add a public getter but no public setter.

Reply:
If this is the case, I will give up changing it. Because it is a simple getting (although it has no setting).

Reply:
The naming is important and it's better to use `closed` rather than `is_closed` (there are only 2 properties named like this but 11 that are named `closed` in the code base).

Reply:
In addition, in the future, please run the tests locally (not necessarily the entire test suite, just the test that you added) before running them on the CI since this could save resources in case your tests fail.

Reply:
I called it `is_closed` because there is also an `is_alive` in the class. Also, closed is really similar to close, which can prevent users from accidentally typing

Reply:
`is_closed` as a method is mainly used by `asyncio` but the multiprocessing module seems to prefer `closed` as a simple property (at least for the Connection class). So I think it makes sense to keep it for the Process class. I'll let @gpshead decide on this one (so do not mark this discussion as resolved for now please).

Reply:
I've made changes.